commons.gradle

```diff
@@ -7,12 +7,12 @@ ext {
     extAgpVersion="4.2.1"
     extCompileSdkVersion=32
```

Comment:
U should update extCompileSdkVersion=32 instead of 
<img width="803" alt="Screen Shot 2022-09-21 at 10 27 56 AM" src="https://user-images.githubusercontent.com/7063654/191571539-85d5e543-70cb-47a5-9dd8-12598f1be4e5.png">

Reply:
The idea was to only update this sample to andriod 13 (API Level 33), because it uses new features which require this version. @v-kpheng we already checked for deprecated APIs for android 13. Can we move all samples to latest android release?

Reply:
@v-kpheng If u have missed it.

Reply:
@chetanvangadiTokbox, you're suggesting that we update to 32 instead of 33?  Since Android 13 was released, shouldn't we target that instead?  Reason being, goal is to ensure that our sample apps target whatever the latest OS version is.  WDYT?

Please advise.  Thanks! 🙏

Reply:
Yes, I agree. My concern was that at the time we hand't completely verified if our APIs were fully compatible with Android 13. With the tests and deprecated APIs checked, I think we feel confortable using it now.